modules/exploits/linux/http/zencart_authenticated_rce.rb

```diff
@@ -0,0 +1,214 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+###
+#
+#
+# This exploit  write payload in database and trig to command
+# a bug in an zencart v1.7.5b web application
+#
+###
+class MetasploitModule < Msf::Exploit::Remote
+    Rank = NormalRanking
+  
+    include Msf::Exploit::Remote::HttpClient
+    include Msf::Exploit::Remote::HttpServer
+  
+    def initialize(info = {})
+      super(
+        update_info(
+          info,
+          'Name'           => 'zencart authenticated remote code execution exploit',
```

Comment:
Title should be capitalized. "Exploit" is implied.

```suggestion
          'Name'           => 'Zencart Authenticated Remote Code Execution',
```